prql-compiler/src/ast/expr.rs

```diff
@@ -60,6 +60,46 @@ pub enum ExprKind {
 /// A name. Generally columns, tables, functions, variables.
 pub type Ident = String;
 
+// We are moving `Ident` from being a string to containing the structure of a
+// namespace. Eventually we can remove this and just use `Ident` everywhere.
+// https://github.com/prql/prql/issues/1031
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
+pub struct IdentWithNamespace {
+    pub namespace: Option<String>,
+    pub ident: Ident,
```

Comment:
I think we need a better name for this. Ident is the whole node, an identifier, while this field is the specific name in a namespace. I used "variable name" or "var_name", but I'm open to suggestions.

Reply:
Yes agree. `var_name` is fine; how about just `var`?

Reply:
`var` it is